testing/zerotier-one/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Kyle Parisi <kyleparisi@gmail.com>
+# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+pkgname=zerotier-one
+pkgver=1.1.14
+pkgrel=0
+pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+url="https://www.zerotier.com/"
+arch="all"
+license="GPL3"
+depends=""
+depends_doc=""
+makedepends="nodejs linux-headers"
+install=""
+subpackages="$pkgname-doc"
+source="
+    $pkgname-$pkgver.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
+    zerotier-one.initd
+    "
+
+builddir="$srcdir"/ZeroTierOne-$pkgver
+build() {
+    cd "$builddir"
+    sed -i.bak s:/bash:/sh:g ./doc/build.sh
+    make || return 1
+}
+
+package() {
+
+    cd "$builddir"
+
+    # setup init script
+    install -m755 -D "$srcdir"/$pkgname.initd \
+            "$pkgdir"/etc/init.d/$pkgname || return 1
+
+    # link all the binaries
+    make DESTDIR="$pkgdir" install || return 1
+}
+
+md5sums="5e381f0864797886b3b3bf20beb49bba  zerotier-one-1.1.14.tar.gz
+4a3fa1f7756633236dc8e2d123393e69  zerotier-one.initd"
+sha256sums="d206069ad21c665159cdececb0a20a21758849ad73d91234d709962b26f634af  zerotier-one-1.1.14.tar.gz
+2d6f929de971f191d819ea94b5d3640de688012d1ddf70e39909746f07ac3fd9  zerotier-one.initd"
+sha512sums="96806a5eab8204d01585c88dba426fb7d47337562074488f87eb81b7ece6643216e2f60adcf1677f9a61fa5ab5187b7569fc1b9565d7055ba106139b6363e7a5  zerotier-one-1.1.14.tar.gz
+073627cdd27a60c487d7d525eabfcfa8f7c1a90b3588aaceace993d0f0ee00cd451033741c66db850ac88c52e1a2f790fccd65edc534aaa09b1a557c57561f80  zerotier-one.initd"
```

Comment:
Please use tabs for indentation, not spaces. In the case of shell scripts, it’s not “just” a matter of code-style, some features does not work with spaces.